doc/source/installation/windows_wsl.rst

```diff
@@ -297,7 +301,7 @@ Where ``$numprocs`` corresponds to the number of processors used for the compila
 Installing Lethe (Step #2)
 -------------------------------------
 
-1. |linux_shell| Set-up the folder structure. Create the ``Software`` folder (if you are doing the candi installation, this folder should alredy exist from Step #1).
+1. |linux_shell| Set-up the folder structure. Create the ``Software`` folder (if you are doing the candi installation, this folder should already exist from Step #1).
```

Comment:
The only thing I don't like is that Software here is with a capital letter. Haha. Otherwise, chill